stores/toml/writer.go

```diff
@@ -0,0 +1,120 @@
+package toml //import "go.mozilla.org/sops/stores/toml"
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/BurntSushi/toml"
+	"go.mozilla.org/sops"
+)
+
+func printTreeBranchInTOML(w io.Writer, branch sops.TreeBranch) error {
+	return printTreeBranch(newIndenter(w, ""), branch)
+}
+
+func printTreeItemInTOML(w io.Writer, item sops.TreeItem) error {
+	return printTreeItem(newIndenter(w, ""), item)
+}
+
+// TOML indenter.
```

Comment:
Documenting why this is needed would be nice.

(not sure it's actually needed -- if it's just for better styled output, mentioning it would be good too)

Reply:
Cool, will do. It all comes down to limitations of the TOML libraries, I'll explain further in the comment.

Reply:
will do